plutus-core/plutus-core/src/PlutusCore/Default/Universe.hs

```diff
@@ -121,14 +119,14 @@ instance GShow DefaultUni where gshowsPrec = showsPrec
 instance Show (DefaultUni a) where
     show DefaultUniInteger             = "integer"
     show DefaultUniByteString          = "bytestring"
-    show DefaultUniChar                = "char"
+    show DefaultUniText                = "text"
```

Comment:
I think we should continue to print it as "string", i.e. it's still the plutus core "string type", it's just backed by the Haskell type `Text`.